packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-bvi-contents.ts

```diff
@@ -0,0 +1,92 @@
+import { localize } from '@deriv/translations';
+import { TJurisdictionCardItems } from 'Components/props.types';
+
+export const jurisdictionBviContents = (): TJurisdictionCardItems => ({
+    is_over_header_available: false,
+    header: localize('British Virgin Islands'),
+    synthetic_contents: [
+        {
+            key: 'assets',
+            title: `${localize('Assets')}`,
```

Comment:
U need not wrap the localize function in a template string here. The function will return a text that gets binded to the prop